### DISCUSSION TITLE
json object key as column value

### DISCUSSION THREAD

**Discussion by medaramesh** [Comcast/react-data-grid#2625] [Q&A]
Hi Guys I am new react-data-grid, just started using. I have data some thing like
{
name: Itemname
itemdesc: itemdesc
price:{
currency:dollar
amount: 9.99
}
}

When I use columns, name, itemdesc is working fine. I can see the data. I lost when I need to show the amount as column and row data. How do I give use Key in column for price.amount ?
Appreciate any suggestions. Thanks

**Comment by mteason** [comment]
Maybe you should remove the price. The nested json is not work for column key/value mapping.

**Comment by andrew0687** [comment]
Hi. Use cell formatter
